src/components/game/GameCanvas.js

```diff
@@ -73,9 +81,36 @@ const GameCanvas = (props) => {
       onMouseMove={handleMouseMove}
       onMouseDown={handleMouseDown}
     >
+
       <canvas id="gameGraphics" width={TOTAL_WIDTH} height={TOTAL_HEIGHT}>
         <GameGraphics gameState={gameState} />
       </canvas>
```

Comment:
Take some time to reorganize/refactor GameCanvas now that it is much more than just a wrapper for the game.